resistbot-node-backend/src/dmHandlers.ts

```diff
@@ -0,0 +1,213 @@
+import { response } from "express";
+import path from "path";
+import urlPackage from "url";
+
+const RP_SCHEME = process.env.RP_SCHEME || "https";
+const RP_NETLOC = process.env.RP_NETLOC || "";
+const RP_BASEPATH = process.env.RP_BASEPATH || "";
+const DISCORD_APP_ID = process.env.DISCORD_APP_ID || "";
+const DISCORD_BOT_TOKEN = process.env.DISCORD_BOT_TOKEN || "";
+
+console.log(
+  `Config is\n RP_SCHEME: ${RP_SCHEME}\n RP_NETLOC: ${RP_NETLOC}\n RP_BASEPATH: ${RP_BASEPATH}\n DISCORD_APP_ID: ${DISCORD_APP_ID}\n DISCORD_BOT_TOKEN: ${DISCORD_BOT_TOKEN}`
+);
```

Comment:
Pretty soon we should either switch to something like Bunyan, or replace entirely with Honeycomb/OpenTelemetry (Sentry is probably worth a try for all of it since it's a clean slate, but we'd need to forward to Senty and Honeycomb. We can discuss as a group more about this later.